tensorflow/python/ops/math_ops.py

```diff
@@ -830,6 +831,8 @@ def to_float(x, name="ToFloat"):
   Raises:
     TypeError: If `x` cannot be cast to the `float32`.
   """
+  if x.dtype.is_complex:
+    logging.warn('Casting complex to real discards imaginary part.')
```

Comment:
The rest of this file uses double quotes. Using single quotes here will cause pylint errors. Please fix.

Reply:
Fixed.